src/ATE_spyder/ate_spyder/widgets/main_widget.py

```diff
@@ -99,6 +107,8 @@ def run_ate_project(self):
         """For now the run of an ATE project is not integrated in the global
         run button yet"""
         pass
+        # self.control(self.project_info)
```

Comment:
So in order to integrate I have to uncomment this and the next line? Or did you forget to remove the commented lines?